src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/ModuleVelocity.kt

```diff
@@ -40,6 +41,9 @@ import net.minecraft.network.packet.s2c.play.EntityVelocityUpdateS2CPacket
 import net.minecraft.network.packet.s2c.play.ExplosionS2CPacket
 import net.minecraft.network.packet.s2c.play.PlayerPositionLookS2CPacket
 
+internal typealias HypixelNoFall = net.ccbluex.liquidbounce.features.module.modules.player.nofall.modes.Hypixel
```

Comment:
Instead of using the typealias HypixelNoFall, might rename the Nofall Mode to NoFallHypixel. If the other modes do not match the naming scheme, rename them as well.